src/gadgets/ecc.rs

```diff
@@ -474,6 +474,7 @@ mod tests {
     }
 
     #[test]
+    #[ignore = "cause it takes a few minutes to run"]
```

Comment:
Okay for now, for the future, I've created an issue #60

Reply:
sounds good